tests/sniffer.py

```diff
@@ -41,6 +42,23 @@ def start(self, test_name):
             err("tcpdump terminated")
             self.tcpdump_proc = None
 
+    def get_packet_capture(self, test_name):
+        if test_name in self.captures:
+            return self.captures[test_name]
+        tshark_command = ['tshark', '-r', self.outputfiles[test_name], '-T', 'json']
+        self.tshark_proc = subprocess.run(
+            tshark_command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        try:
+            capture = json.loads(self.tshark_proc.stdout)
+            def ieee_filter(x): return 'ieee1905' in x['_source']['layers']
```

Comment:
I don't think this is appropriate - instead, we should capture only ieee1905 packets from the start when we use tcpdump:
```
tcpdump -i<iface> ether proto 0x893a
```